### ISSUE TITLE
Got an error for 'farver' when using 'ggplot2'

### ISSUE THREAD

**Issue by Sunnyy651** [thomasp85/farver#29]
Hi, 
I have encountered this issue on my working computer when I was trying to plot using ggplot2 (please see below). I can't get any result when googling  this error so I think it is worthwhile to write a new post. Thanks for helping me out!

> library(dplyr)
> library(ggplot2)

> annoying <- tibble(
+   `1` = 1:10,
+   `2` = `1` * 2 + rnorm(length(`1`))
+ )
> ggplot(annoying, aes(x = `1`, y = `2`)) +
+   geom_point()
Error: .onLoad failed in loadNamespace() for 'farver', details:
  call: .Call("load_colour_names_c", c(all_colours, as.character(seq_along(def_palette) - 
  error: "load_colour_names_c" not available for .Call() for package "farver"

**Comment by thomasp85** [comment]
Hmm... it appears something is messed up with your installation of farver - have you tried reinstalling it?

**Comment by Sunnyy651** [comment]
Yes, I tried remove and reinstall it several times but it didn't help. I have to mention that my 'farver' package and all dependencies are installed in a location where I have permission to make changes, but 'ggplot2' is preinstalled in drive C: where I do not have permission to make any change (as I am using a corporate device). Will that be the reason why my package all got messed up?

**Comment by thomasp85** [comment]
Sorry for never returning to this issue. Is this still a problem?
